src/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,236 @@
+  defaultIsOpen?: boolean,
+  /** @Private Tooltips cannot be disabled and this prop will be stripped */
+  disabled?: boolean,
+  /** @Private Tooltips must have arrows according to design spec */
+  hasArrow?: boolean,
+  /** For use with controlled components, in which the app manages Tooltip state */
+  isOpen?: boolean,
+  /** Called when Tooltip is closed */
+  onClose?: (event: SyntheticEvent<>) => void,
+  /** Called when Tooltip is opened */
+  onOpen?: (event: SyntheticEvent<>) => void,
+  /** Placement of the Tooltip */
+  placement?:
+    | 'auto'
+    | 'auto-end'
+    | 'auto-start'
+    | 'bottom'
+    | 'bottom-end'
+    | 'bottom-start'
+    | 'left'
+    | 'left-end'
+    | 'left-start'
+    | 'right'
+    | 'right-end'
+    | 'right-start'
+    | 'top'
+    | 'top-end'
+    | 'top-start',
+  /** @Private Tooltips should not have subtitles and will be removed */
+  subtitle?: any,
+  /** @Private Tooltips should not have titles and will be removed */
+  title?: any,
+};
+
+type State = {
+  isOpen?: boolean
+};
+
+const delay = 200; //ms
+
+export const componentTheme = (baseTheme: Object) => {
+  return {
+    ...mapComponentThemes(
+      {
+        name: 'Popover',
+        theme: popoverComponentTheme(baseTheme)
+      },
+      {
+        name: 'Tooltip',
+        theme: {
+          TooltipArrow_backgroundColor: baseTheme.color_gray_90,
+          TooltipArrow_borderColor: baseTheme.color_gray_90,
+          TooltipContent_backgroundColor: baseTheme.color_gray_90,
+          TooltipContent_borderColor: baseTheme.color_gray_90,
+          TooltipContent_color: baseTheme.color_white,
+          TooltipContent_maxWidth: '18em',
+          TooltipContentBlock_marginVertical: '0',
+          TooltipContentBlock_paddingHorizontal: baseTheme.space_inset_md,
+          TooltipTriggerText_borderStyle: 'dashed',
+          TooltipTriggerText_borderColor: baseTheme.color_text,
+          TooltipTriggerText_borderWidth: '1px'
+        }
+      },
+      baseTheme
+    )
+  };
+};
+
+const Root = createThemedComponent(Popover, ({ theme: baseTheme }) => {
+  return {
+    ...mapComponentThemes(
+      {
+        name: 'Tooltip',
+        theme: componentTheme(baseTheme)
+      },
+      {
+        name: 'Popover',
+        theme: {}
+      },
+      baseTheme
+    )
+  };
+});
+
+const TriggerText = createStyledComponent('span', ({ theme: baseTheme }) => {
+  const theme = componentTheme(baseTheme);
+
+  return {
+    borderBottomStyle: theme.TooltipTriggerText_borderStyle,
+    borderBottomColor: theme.TooltipTriggerText_borderColor,
+    borderBottomWidth: theme.TooltipTriggerText_borderWidth
+  };
+});
+
+/**
+ * Tooltips hold supporting information for user controls.
+ * Tooltips float over page content.
+ * You can control placement and open/close actions to customize your implementation.
+ * Tooltips can be toggled on a user action or a state change.
+ */
+export default class Tooltip extends Component<Props, State> {
```

Comment:
I would expect Tooltip default `placement` to be `top`.

Reply:
most other design systems that I looked at had bottom as the default. Because of that and Popover's default being 'bottom' I just left it as popover's default.